models/pancreas_ct_dints_segmentation/scripts/search.py

```diff
@@ -28,7 +28,7 @@
 from monai.bundle import ConfigParser
 from monai.data import ThreadDataLoader, partition_dataset
 from monai.inferers import sliding_window_inference
-from monai.metrics import compute_meandice
+from monai.metrics import compute_dice
```

Comment:
update the API, now `monai.metrics` does not have `compute_meandice`
cc @dongyang0122